core/src/avm2/globals/flash/display/interactiveobject.rs

```diff
@@ -141,7 +141,8 @@ fn set_context_menu<'gc>(
         .and_then(|t| t.as_display_object())
         .and_then(|dobj| dobj.as_interactive())
     {
-        let cls = activation.avm2().classes().nativemenu;
+        let cls_name = QName::new(Namespace::package("flash.display"), "NativeMenu");
+        let cls = activation.resolve_class(&cls_name.into())?;
```

Comment:
I think `flash.display.NativeMenu` can be added to the `avm2_system_classes_playerglobal!` macro invocation in `globals.rs`, so `activation.context.avm2.classes().nativemenu` will remain.

Reply:
I explicitly changed it this way, as I don't think we should clutter `SystemClasses` every time we need to refer to a some builtin class in not-perf-critical code. Further, rewriting `InteractiveObject` to AS would make this type check/coercion automatic (at least that's my understanding).

Reply:
It just feels to me a good practice to keep everything in one place, but since we anyway talk about to-be-rewritten code, I guess this is fine.

Reply:
> keep everything in one place

It's the other way around - `SystemClasses` is the "redundant data duplication" here. The more AS classes we add, the less (percentage-wise) classes we'll need there; long-term it might end up with mostly just the primitives' classes.